survey/models/survey.py

```diff
@@ -14,12 +14,24 @@ def in_duration_day():
 
 class Survey(models.Model):
 
+    ALL_IN_ONE_PAGE = 0
+    BY_QUESTION = 1
+    BY_CATEGORY = 2
+
+    DISPLAY_METHOD_CHOICES = [
+        (BY_QUESTION, "By question"),
+        (BY_CATEGORY, "By category"),
```

Comment:
I think we need to translate the values here ?
```suggestion
        (BY_CATEGORY, _("By category")),
```

Reply:
I've updated the three, and given approximate translations for languages I have reasonable knowledge of or had a native speaker handy.

Reply:
Nice. Too bad we know exactly the same one ;) (Ton français est très correct, c'est juste un peu bizarre d'être vouvoyer sur internet :) )

Reply:
Ca fait un siècle que j'ai pas écrit en français sur internet... et les 4 dernières années au taffe sur Paname je devais tout écrire en anglais, donc je commence à perdre les bonnes habitudes :-(

Reply:
C'est pourtant simple le vouvoiement/tutoiement :D (No : https://www.latimes.com/opinion/op-ed/la-og-bastile-vous-tu-20140711-htmlstory.html)
Et j'ai oublié de dire que tu peux également t'ajouter dans la partie langage du Readme :)

Reply:
Bah, justement, I wasn't feeling lucky :). Et je m'ajoute !